app/controllers/case_information_controller.rb

```diff
@@ -10,7 +10,50 @@ def new
     )
   end
 
-  def edit; end
+  def create
+    @case_info = CaseInformation.create(
+      nomis_offender_id: case_information_params[:nomis_offender_id],
+      tier: case_information_params[:tier],
+      case_allocation: case_information_params[:case_allocation],
+      probation_service: case_information_params[:probation_service],
+      last_known_location: case_information_params[:last_known_location],
+      team_id: team_identifier,
+      manual_entry: true
+    )
+
+    params[:stage] == 'last_location' ? handle_stage1 : handle_stage2
+
+    if @case_info.valid?
+      welsh_offender?(probation: @case_info.probation_service)
+      @case_info.save
+      send_email
+      redirect_to prison_summary_pending_path(active_prison_id, sort: params[:sort], page: params[:page])
+    end
+  end
+
+  def edit
+    @case_info.last_known_location = if @case_info.probation_service == 'England'
+                                       'No'
+                                     else
+                                       'Yes'
+                                     end
+    @team_name = Team.find_by(id: @case_info.team_id)&.name
+  end
+
+  def update
+    @case_info = CaseInformation.find_by(nomis_offender_id: case_information_params[:nomis_offender_id])
+    @prisoner = prisoner(case_information_params[:nomis_offender_id])
+
+    # we only send email if the ldu is different from previous
```

Comment:
IMHO there is a more idiomatic way of implementing this. 
If the controller just updates model attributes (via model[:attr] = value) rather than calling update, then the changes will not be saved (yet). Then the controller can look at the 'team_was' attribute to determine the old team value (and hence the old LDU value)
P.S. it is a little un-idiomatic to have a method ending with a ? peform a side effect.